cli/commands/analyze-loading-optimization.mjs

```diff
@@ -0,0 +1,527 @@
+/**
+ * CLI command to analyze several URLs for initial-viewport image optimization issues.
+ *
+ * WPP Research, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import puppeteer, { Browser } from 'puppeteer';
+import round from 'lodash-es/round.js';
+
+/* eslint-disable jsdoc/valid-types */
+/** @typedef {import("web-vitals").LCPMetricWithAttribution} LCPMetricWithAttribution */
+/* eslint-enable jsdoc/valid-types */
```

Comment:
Do we really need this? As far as I can tell this is the only reason that you added the `web-vitals` dependency to `package.json`, which to me is not useful enough. Especially since then the `jsdoc/valid-types` rule gets disabled for the line anyway.

Can't we work without the concrete `@typedef` here, given we don't actually need the `web-vitals` dependency (since it's only injected into the browser from CDN anyway)?

Reply:
Yes, the benefit of the `web-vitals` dependency is for the typing.

 For example, when we are working with `webVitalsLCP`:

![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/b05b257f-d65f-4e51-b36f-aee5d6748742)

Then the editor can know that `webVitalsLCP.attribution` is is an `LCPAttribution` object:

![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/ffa7f7c7-daa1-45d8-a31f-9c369174dc7d)

That its `lcpEntry` property is a `LargestContentfulPaint` object:

![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/735ed7f8-f2ba-472f-8e3b-e6784edd7a51)

And that it has an `element` property which is an `Element` object:

![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/6b5a8722-78bb-4d3b-b4f1-8cfebe17a8e7)

We get autocompletion as well:

![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/0a66329c-1a07-4b30-9c46-5576890eaff3)

Without the `web-vitals` types, then the editor doesn't know what properties the object has:

![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/9ed1a407-5548-4f6b-971e-608476e66530)